js/server/tests/resilience/resilience-synchronous-repl-cluster.js

```diff
@@ -321,6 +322,26 @@ function SynchronousReplicationSuite () {
 ////////////////////////////////////////////////////////////////////////////////
 
     setUp : function () {
+      if (!inquiry_checked) {
+        console.warn("Checking inquiry");
+        var writeResult = ArangoAgency.write([[{"a":1},{"a":{"oldEmpty":true}},"INTEGRATION_TEST_INQUIRY_ERROR_503"]]);
+        console.log("Inquired successfully a matched precondition under 503 response from write");
+        try {
+          writeResult = ArangoAgency.write([[{"a":1},{"a":0},"INTEGRATION_TEST_INQUIRY_ERROR_503"]]);
+          assertEqual(true,false);
+        } catch (e1) {
+          console.log("Inquired successfully a failed precondition under 503 response from write");
```

Comment:
I think we should test for the expected error code here.
And not have console output.